admin/app/mirage/data/apps.js

```diff
@@ -1,7 +1,7 @@
 export default [
   {
     id: 1,
-    name: 'blog',
+    name: 'Blog',
```

Comment:
is this representative of what will come over from the backend? i believe this is from the rails router code, i.e.

```rb
front_end :blog
```